js/assessments/sentenceLengthInDescriptionAssessment.js

```diff
@@ -0,0 +1,55 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var countTooLongSentences = require( "./checkForTooLongSentences.js" );
+
+/**
+ * Calculates sentence length score
+ * @param {array} sentences The array containing sentences.
+ * @param {object} i18n The object used for translations.
+ * @returns {object} Object containing score and text.
+ */
+var calculateSentenceLengthResult = function( sentences, i18n ) {
```

Comment:
The same feedback given for `sentenceLenghtInTextAssessment.js`, also applies here.